packages/protocol/contracts/bridge/IBridge.sol

```diff
@@ -8,18 +8,17 @@ pragma solidity ^0.8.20;
 
 /// @title IBridge
 /// @notice The bridge used in conjunction with the {ISignalService}.
-/// @dev Ether is held by Bridges on L1 and by the EtherVault on L2,
-/// not by token vaults.
+/// @dev Ether is held by Bridges on L1 and L2s.
 interface IBridge {
     struct Message {
         // Message ID.
         uint128 id;
         // Message sender address (auto filled).
         address from;
         // Source chain ID (auto filled).
-        uint256 srcChainId;
+        uint64 srcChainId;
         // Destination chain ID where the `to` address lives (auto filled).
-        uint256 destChainId;
+        uint64 destChainId;
```

Comment:
`user` seems a bit like a strange name for the use of this variable if I understand correctly. Maybe `processor` or something would be closer to the actual use?

Reply:
The 'user' is actually the msg.sender on the source chain to send a message, so maybe 'owner' is a better name? I made the change, let me know if you like it.

Reply:
message.from is the processor on the source chain; message.to is the process on the dest chain.

Reply:
Oh yeah owner is a good name!

> The 'user' is actually the msg.sender on the source chain to send a message, so maybe 'owner' is a better name? I made the change, let me know if you like it.

It's not necessarily `msg.sender` when using the bridge directly right?

> message.from is the processor on the source chain; message.to is the process on the dest chain.

I see `message.from` is kind of the simulated `msg.sender` I think, with `message.user` having some special processing powers. But agreed owner is a better name!